packages/workflow/src/workflow.ts

```diff
@@ -1335,7 +1349,20 @@ export function setDefaultSignalHandler(handler: DefaultSignalHandler | undefine
  * });
  * ```
  *
- * would result in the Workflow having these Search Attributes:
+ * Equivalently, upserting a workflow's search attributes using SearchAttributeUpdatePair[]:
+ *
+ * ```ts
+ * upsertSearchAttributes([
+ *  { key: { name: 'CustomIntField', type: 'INT' }, value: 1 },
```

Comment:
Please add a `defineSearchAttribute()` helper function in the commmon package, and use it here to predefine the SAs rather than inlining the definition as the value of the `key` property. That's the usage pattern that users are expected to follow.

Reply:
Done.